server/app/Http/Controllers/UsersController.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\UsersRequest\UserCreateRequest;
+use App\Http\Requests\UsersRequest\UserUpdateRequest;
+use App\User;
+use Hash;
+
+class UsersController extends Controller
+{
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+
+        $campos = ['id', 'name', 'email', 'created_at'];
+
+        $porPag = $request->input("porPag");
```

Comment:
El acceso a los parámetros que se reciben en la `request` (línea 24) debería ser movido a la línea 21, para diferenciar más facilmente el input del resto del código.

Reply:
Creo que es mejor si evitamos las abreviaciones, a menos que la palabra sea excesivamente larga
```suggestion
        $porPagina = $request->input("porPagina ");
```

Reply:
Solucionado en 43799df8d31d0cd73aa7b0a122c103e7fd0eaacc ✔